sema4ai/src/sema4ai_code/robocorp_language_server.py

```diff
@@ -1798,12 +1798,22 @@ def _agent_cli_download_threaded(
         return {"success": True, "message": None, "result": location}
 
     @command_dispatcher(commands.SEMA4AI_ACTION_SERVER_VERSION_INTERNAL)
-    def _action_server_version(self) -> ActionResultDict:
-        return self._action_server.get_version().as_dict()
+    def _action_server_version(self, download_if_missing=True) -> ActionResultDict:
+        return self._action_server.get_version(
+            download_if_missing=download_if_missing
+        ).as_dict()
 
     @command_dispatcher(commands.SEMA4AI_AGENT_CLI_VERSION_INTERNAL)
-    def _agent_cli_version(self) -> ActionResultDict:
-        return self._agent_cli.get_version().as_dict()
+    def _agent_cli_version(self, download_if_missing=True) -> ActionResultDict:
+        return self._agent_cli.get_version(
+            download_if_missing=download_if_missing
+        ).as_dict()
+
+    def m_action_server_version(self, download_if_missing=True) -> ActionResultDict:
```

Comment:
I know we talked about preferring this construct, but I think we shouldn't really duplicate (i.e.: we can refactor everywhere to use the new construct, but otherwise I think just keep using the existing action -- for new things sure, the new construct is preferred for internal things).